swapper/__init__.py

```diff
@@ -43,14 +43,17 @@ def get_model_name(app_label, model):
     return is_swapped(app_label, model) or join(app_label, model)
 
 
-def dependency(app_label, model):
+def dependency(app_label, model, latest=False):
     """
     Returns a Django 1.7+ style dependency tuple for inclusion in
     migration.dependencies[]
     """
     from django.db.migrations import swappable_dependency
 
-    return swappable_dependency(get_model_name(app_label, model))
+    dependencies = swappable_dependency(get_model_name(app_label, model))
+    if not latest:
+        return dependencies
+    return dependencies[0], '__latest__'
```

Comment:
we need a test for this line, please:
https://github.com/openwisp/django-swappable-models/blob/master/tests/test_swapper.py#L78-L82